### DISCUSSION TITLE
lazy loading when an event is fired

### DISCUSSION THREAD

**Discussion by gsaple** [wbthomason/packer.nvim#1129] [General]
Say I set up a lazy loading using `event`
```
use {
  'username/cool_plugin', 
  ...
  event = 'my_event' (e.g. InsertEnter)
}
```
my question: does the `cool_plugin` loaded **only once** when `my_event` fired,  or,  does the  `cool_plugin` loaded **every time** when `my_event` fired? 
Thank you!

**Comment by yutkat** [comment]
It loads `only once`. 

You can check this compiled configuration file. https://github.com/wbthomason/packer.nvim/blob/6afb67460283f0e990d35d229fd38fdc04063e0a/lua/packer.lua#L15

You can see that `++once` is written.
```lua
vim.cmd [[au WinNew * ++once lua require("packer.load")({'bufresize.nvim'}, { event = "WinNew *" }, _G.packer_plugins)]]
```
